boards/DCM/Src/App/states/App_FaultState.c

```diff
@@ -23,10 +23,11 @@ static void FaultStateRunOnExit(struct StateMachine *const state_machine)
 const struct State *App_GetFaultState(void)
 {
     static struct State fault_state = {
-        .name         = "FAULT",
-        .run_on_entry = FaultStateRunOnEntry,
-        .run_on_tick  = FaultStateRunOnTick,
-        .run_on_exit  = FaultStateRunOnExit,
+        .name             = "FAULT",
+        .run_on_entry     = FaultStateRunOnEntry,
+        .run_on_tick_1Hz = FaultStateRunOnTick,
```

Comment:
nit; for this PR can we leave the `OnTick` functions as 1kHz rather 1Hz? I believe all the state machines are ticking at 1kHz at the moment

Reply:
sure, no problem